src/Node/Net.purs

```diff
@@ -0,0 +1,585 @@
+module Node.Net
+  ( Address
+  , ConnectOptions
+  , ListenOptions
+  , Lookup
+  , Server
+  , ServerOptions
+  , Socket
+  , SocketOptions
+  , address
+  , allowHalfOpen
+  , backlog
+  , bufferSize
+  , bytesRead
+  , bytesWritten
+  , close
+  , connect
+  , connectICP
+  , connectTCP
+  , connecting
+  , createConnection
+  , createConnectionICP
+  , createConnectionTCP
+  , createServer
+  , destroy
+  , destroyed
+  , end
+  , endString
+  , exclusive
+  , family
+  , fd
+  , getConnections
+  , hints
+  , host
+  , ipv6Only
+  , isIP
+  , isIPv4
+  , isIPv6
+  , listen
+  , listenICP
+  , listenTCP
+  , listening
+  , localAddress
+  , localAddressOption
```

Comment:
Most of the other options use the name of the field (like `fd` and `host` and whatever), since there's already a `localAddress` value, this had to be a different name. Is this an okay suffix? Should we suffix all options? Should we move the options elsewhere? Should we rename `localAddress` to something and make this option called `localAddress`? Should we do something else?

Reply:
How about having a prefix on all option values which depends on the type of `Options` we are dealing with? So for example we'd have `connectLocalAddress` and `connectLocalPort`, but also `connectFamily`, `connectPort`, `connectHost`, and so on (for uniformity).

Reply:
Works for me.